packages/golub/example/README.md

```diff
@@ -1,30 +1,30 @@
 <?code-excerpt path-base="app"?>
-# Pigeon Examples
+# Golub Examples
```

Comment:
and again – lots of references to pigeon

Reply:
https://github.com/Yobari-Timeliners/golub/issues/105